docs/api/classifier/rocauc.rst

```diff
@@ -9,38 +9,29 @@ The Receiver Operating Characteristic (ROC) is a measure of a classifier's predi
 
 This leads to another metric, area under the curve (AUC), which is a computation of the relationship between false positives and true positives. The higher the AUC, the better the model generally is. However, it is also important to inspect the "steepness" of the curve, as this describes the maximization of the true positive rate while minimizing the false positive rate.
 
-.. code:: python
+.. plot::
+    :context: close-figs
 
     from sklearn.model_selection import train_test_split
+    from yellowbrick.classifier import ROCAUC
+    from sklearn.linear_model import LogisticRegression
+    from yellowbrick.datasets import load_occupancy
 
     # Load the classification data set
-    data = load_data("occupancy")
+    X, y = load_occupancy()
 
-    # Specify the features of interest and the classes of the target
-    features = ["temperature", "relative humidity", "light", "C02", "humidity"]
+    # Specify the classes of the target
     classes = ["unoccupied", "occupied"]
 
-    # Extract the instances and target
-    X = data[features]
-    y = data.occupancy
-
     # Create the train and test data
     X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)
 
-.. code:: python
-
-    from yellowbrick.classifier import ROCAUC
-    from sklearn.linear_model import LogisticRegression
-
     # Instantiate the visualizer with the classification model
     visualizer = ROCAUC(LogisticRegression(), classes=classes)
 
     visualizer.fit(X_train, y_train)  # Fit the training data to the visualizer
     visualizer.score(X_test, y_test)  # Evaluate the model on the test data
-    g = visualizer.poof()             # Draw/show/poof the data
-
-
-.. image:: images/rocauc_binary.png
+    visualizer.poof()             # Draw/show/poof the data
 
 
 .. warning::
```

Comment:
It's just occurring to me that now that v0.9 is out we might want to update this to say something like:


Versions of Yellowbrick =< v0.8 had a `bug <https://github.com/DistrictDataLabs/yellowbrick/blob/develop/examples/rebeccabilbro/rocauc_bug_research.ipynb>`_ that triggered an ``IndexError`` when attempting binary classification using a Scikit-learn-style estimator with only a ``decision_function``.  This has been fixed as of v0.9, where the ``micro``,
    ``macro``, and ``per-class`` parameters of ``ROCAUC`` are set to ``False`` for such classifiers.